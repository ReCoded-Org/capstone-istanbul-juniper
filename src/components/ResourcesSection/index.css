.resourceCardWrapper {
  background-color: #b4d0c4;
  padding: 20px 0;
}

.resourceCardWrapper__cardImage img {
  width: 100%;
}

.resourceCardWrapper__cardName a {
  color: black;
}

.resourceCardWrapper__cardName {
  margin-bottom: 10px;
  font-family: Salsa;
  font-weight: bold;
  font-size: 20px;
  text-align: center;
}
