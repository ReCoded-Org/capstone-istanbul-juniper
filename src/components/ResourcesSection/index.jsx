import React from "react";
import { Col, Row } from "antd";
import "./index.css";
import "antd/dist/antd.css";
import { Link } from "react-router-dom";
import { useTranslation } from "react-i18next";
import recycleImg from "./../../images/recycleResourceSetion.svg";
import earthImg from "./../../images/earthResourceSetion.svg";
import pollutionImg from "./../../images/pollutionResourceSetion.svg";

const ResourcesSection = () => {
  const [t] = useTranslation();
  return (
    <div className="resourceCardWrapper">
      <Row justify="center" gutter={[16, 16]}>
        <Col>
          <Row justify="center">
            <Col className="resourceCardWrapper__cardName">
              <Link to="/resources">
                {t("home.resourcesSection.pollution")}
              </Link>
            </Col>
          </Row>
          <Row justify="center">
            <Col className="resourceCardWrapper__cardImage">
              <Link to="/resources">
                <img alt="pollution" src={pollutionImg} />
              </Link>
            </Col>
          </Row>
        </Col>
        <Col>
          <Row justify="center">
            <Col className="resourceCardWrapper__cardName">
              <Link to="/resources">
                {t("home.resourcesSection.climateChange")}
              </Link>
            </Col>
          </Row>
          <Row justify="center">
            <Col className="resourceCardWrapper__cardImage">
              <Link to="/resources">
                <img alt="Climate Change" src={earthImg} />
              </Link>
            </Col>
          </Row>
        </Col>
        <Col>
          <Row justify="center">
            <Col className="resourceCardWrapper__cardName">
              <Link to="/resources">
                {t("home.resourcesSection.reduceReuseRecycle")}
              </Link>
            </Col>
          </Row>
          <Row justify="center">
            <Col className="resourceCardWrapper__cardImage">
              <Link to="/resources">
                <img alt="Recycle" src={recycleImg} />
              </Link>
            </Col>
          </Row>
        </Col>
      </Row>
    </div>
  );
};

export default ResourcesSection;
