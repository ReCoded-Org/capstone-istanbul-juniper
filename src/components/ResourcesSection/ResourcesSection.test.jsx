import React from "react";
import renderer from "react-test-renderer";
import ResourcesSection from ".";
import { BrowserRouter as Router } from "react-router-dom";

it("ResourcesSection renders correctly", () => {
  const tree = renderer
    .create(
      <Router>
        <ResourcesSection />
      </Router>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
