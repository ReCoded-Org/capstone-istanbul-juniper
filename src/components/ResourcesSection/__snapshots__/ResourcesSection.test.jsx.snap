// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResourcesSection renders correctly 1`] = `
<div
  className="site-card-wrapper"
>
  <div
    className="ant-row ant-row-center"
    style={
      Object {
        "marginBottom": 8,
        "marginLeft": -8,
        "marginRight": -8,
        "marginTop": -8,
      }
    }
  >
    <div
      className="ant-col ant-col-sm-24 ant-col-xxl-6"
      style={
        Object {
          "paddingBottom": 8,
          "paddingLeft": 8,
          "paddingRight": 8,
          "paddingTop": 8,
        }
      }
    >
      <div
        className="ant-row ant-row-center"
        style={Object {}}
      >
        <div
          className="ant-col ant-col-22 site-card-wrapper__cardName"
          style={Object {}}
        >
          <a
            href="/pollution"
            onClick={[Function]}
          >
            home.resourcesSection.pollution
             
          </a>
        </div>
        <div
          className="ant-col ant-col-22 site-card-wrapper__cardImage"
          style={Object {}}
        >
          <a
            href="/pollution"
            onClick={[Function]}
          >
            <img
              alt="Poulution"
              src="pollution.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      className="ant-col ant-col-sm-24 ant-col-xxl-6"
      style={
        Object {
          "paddingBottom": 8,
          "paddingLeft": 8,
          "paddingRight": 8,
          "paddingTop": 8,
        }
      }
    >
      <div
        className="ant-row ant-row-center"
        style={Object {}}
      >
        <div
          className="ant-col ant-col-22 site-card-wrapper__cardName"
          style={Object {}}
        >
          <a
            href="/climateChange"
            onClick={[Function]}
          >
            home.resourcesSection.climateChange
          </a>
        </div>
        <div
          className="ant-col ant-col-22 site-card-wrapper__cardImage"
          style={Object {}}
        >
          <a
            href="/climateChange"
            onClick={[Function]}
          >
            <img
              alt="Climate Change"
              src="earth.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      className="ant-col ant-col-sm-24 ant-col-xxl-6"
      style={
        Object {
          "paddingBottom": 8,
          "paddingLeft": 8,
          "paddingRight": 8,
          "paddingTop": 8,
        }
      }
    >
      <div
        className="ant-row ant-row-center"
        style={Object {}}
      >
        <div
          className="ant-col ant-col-22 site-card-wrapper__cardName"
          style={Object {}}
        >
          <a
            href="/recycle"
            onClick={[Function]}
          >
            home.resourcesSection.reduceReuseRecycle
          </a>
        </div>
        <div
          className="ant-col ant-col-22 site-card-wrapper__cardImage"
          style={Object {}}
        >
          <a
            href="/recycle"
            onClick={[Function]}
          >
            <img
              alt="Recycle"
              src="recycle.svg"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
